sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -634,6 +634,24 @@ namespace Azure { namespace Core { namespace Http {
     {
     }
 
+    /**
+     * @brief Copy a raw response to construct a new one.
+     *
+     * @remark The body stream won't be copied.
+     *
+     * @param response A reference for copying the raw response.
+     */
+    RawResponse(RawResponse const& response)
```

Comment:
This is used by the LRO.
Once the operation is completed, is keep calling `operation.Value()` we keep returning a copy of the response and the raw response.

Otherwise, the `operation.Value()` would need to be renamed to ExtractValue() and it would work only once. The operation would gave up on the rawResponse (which is a unique_ptr).  
But operation.Value() is define in the Core.Operation base class.  So the contract is currently requiring an operation to support returning a copy of the response.